sli/dictutils.h

```diff
@@ -154,7 +155,7 @@ get_long_in_range( const DictionaryDatum& d, Name const n, long min, long max, i
   // token.
   const Token& t = d->lookup2( n );
   DoubleDatum* dd = dynamic_cast< DoubleDatum* >( t.datum() );
-  long x = 0;
+  long x = std::numeric_limits< long >::quiet_NaN();
```

Comment:
A bit surprised this compiles, given that NaN is not defined for longs, only for floating point types. Here maybe `...::min` is better.

Reply:
Was also wondering, but didn't look further into it, as it compiled without any issue.  
Unsure whether `::min` or simply `0` is more explicit here, but we can just go for `::min` as it at least makes the reader think about it and might come to the conclusion that the `::min` is just a placeholder value.